add: launchpad workshop added (#21)
graphops/docs: docs/mips-resources/workshops/workshop4.md

---
sidebar_position: 6
---

# Workshop 4: Launchpad 101

All the calls and workshops are recorded and can be found in [this](https://www.youtube.com/playlist?list=PLTqyKgxaGF3SvYpAaIFAj9Gr-Rp0l7gUa) playlist. 

In [Indexer Office Hours](https://www.youtube.com/channel/UCQ7G_cCufIVUdUUUf-jdoVA) 80, Ana Calin and Chris Wessels from GraphOps gave a workshop on [Launchpad](https://www.youtube.com/watch?v=v82mKAq46K8&list=PLTqyKgxaGF3SvYpAaIFAj9Gr-Rp0l7gUa&index=6), a tool for orchestrating the indexer stack.

> [Here](https://docs.google.com/presentation/u/1/d/1Pd0RuH8Hwq3Y62fMycM6Z6yW4uYsOWNlGBTZmIh9dTE/edit?usp=sharing) is the slides of the workshop. 

Reviewer: ```suggestion
[Here](https://docs.google.com/presentation/u/1/d/1Pd0RuH8Hwq3Y62fMycM6Z6yW4uYsOWNlGBTZmIh9dTE/edit?usp=sharing) are the slides of the workshop. 
```

Here is what is covered in the workshop: 

![MIPs Workshop #4 Agenda](/img/workshops/mips-workshop4-agenda.png)
## Highlights of the Workshop

**Why Launchpad?**

> Launchpad is designed to meet tomorrow’s needs. It provides a strong foundation for long-term multi-chain indexing and encodes our beliefs and principles about how large-scale infrastructure should be architected and operated.

Reviewer: ```suggestion
Launchpad is designed to meet tomorrow’s needs. It provides a strong foundation for long-term multi-chain indexing and encodes GraphOps' beliefs and principles about how large-scale infrastructure should be architected and operated.
```

![MIPs Workshop #4 Agenda](/img/workshops/mips-workshop4-why-launchpad.png)
       
**What is Launchpad?**

An opinionated set of tools on your local machine, layered over one
another to provide a declarative workflow through all the layers of your infrastructure.

![MIPs Workshop #4 Agenda](/img/workshops/mips-workshop4-what-launchpad.png)

An opinionated stack of software running on your host machines, orchestrated with Kubernetes, providing core cluster functionality, and everything you need to run a scalable Indexer on the Graph’s Decentralised Network. 

![MIPs Workshop #4 Agenda](/img/workshops/mips-workshop4-what-launchpad2.png)

> The core of Launchpad consists of 2 repositories: launchpad-starter and launchpad-core. Another public repository indexers should run is helm-charts.

Reviewer: ```suggestion
The core of Launchpad consists of 2 repositories: [`launchpad-starter`](https://github.com/graphops/launchpad-starter) and [`launchpad-core`](https://github.com/graphops/launchpad-core). It also deploys releases defined at [helm-charts repo](https://github.com/graphops/helm-charts)
```
